# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/neovim/hotfix-no-ccache.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- neovim-0.9.5/CMakeLists.txt.vanilla	2024-01-10 12:20:02.013722896 +0100
+++ neovim-0.9.5/CMakeLists.txt	2024-01-10 12:20:28.099954469 +0100
@@ -30,10 +30,10 @@
 
 set(TOUCHES_DIR ${PROJECT_BINARY_DIR}/touches)
 
-find_program(CCACHE_PRG ccache)
-if(CCACHE_PRG)
-  set(CMAKE_C_COMPILER_LAUNCHER ${CMAKE_COMMAND} -E env CCACHE_SLOPPINESS=pch_defines,time_macros ${CCACHE_PRG})
-endif()
+#find_program(CCACHE_PRG ccache)
+#if(CCACHE_PRG)
+#  set(CMAKE_C_COMPILER_LAUNCHER ${CMAKE_COMMAND} -E env CCACHE_SLOPPINESS=pch_defines,time_macros ${CCACHE_PRG})
+#endif()
 
 if(NOT CI_BUILD)
   set(CMAKE_INSTALL_MESSAGE NEVER)
